src/app/components/page-title.tsx

```diff
@@ -0,0 +1,24 @@
+import { ReactNode } from 'react';
+import { BoxProps } from '@stacks/ui';
+
+import { Title } from '@app/components/typography';
+
+interface PageTitleProps extends BoxProps {
+  children?: ReactNode;
+  isCentered?: boolean;
+  isWelcomePage?: boolean;
```

Comment:
I'd call this something more generic like `isHeadline`. This describes what it does vs. where it's used, and may have uses outside the welcome page.

Reply:
Good call, thx.